pyvcloud/vcd/org.py

```diff
@@ -537,7 +546,7 @@ def get_roles_query(self, name_filter=None):
         org_filter = None
         if self.is_admin:
             resource_type = 'adminRole'
-            org_filter = 'orgName==%s' % self.resource.get('name')
+            org_filter = 'org==%s' % self.resource.get('href')
```

Comment:
what is the reason to change this?

Reply:
Parity with the other areas in code, where we are limiting what the sys admin can see by using href of the org as a filter. The functionality isn't getting impacted, it's mostly cosmetic.